submissions/Liddell777/hooli-style-popup/style.css

```diff
@@ -0,0 +1,277 @@
+* {
+    padding: 0;
+    margin: 0;
+    border: 0;
+}
+
+*, *:before, *:after {
+    box-sizing: border-box;
+}
+
+body{
+    height: 100%;
+    width: 100%;
+    line-height: 1;
+    font-size: 14px;
+    overflow: hidden;
+    font-family: "Titillium Web", sans-serif;
+}
+
+input, button, textarea {
+    font-family: inherit;
+}
+
+input::-ms-clear {
+    display: none;
+}
+
+button {
+    cursor: pointer;
+}
+
+button::-moz-focus-inner {
+    padding: 0;
+    border: 0;
+}
+
+a, a:visited, a:hover  {
+    text-decoration: none;
+}
+
+ul {
+    list-style: none;
+}
+
+img {
+    vertical-align: top;
+    max-width: 100%;
+}
+main {
+    background: 
+    url(../img/bg5.gif) center no-repeat;
+    background-size: 500px;
+}
+
+
+header {
+    width: 100vw;
+    height: 80px;
+} 
+
+.nav {
+    max-width: 90vw;
+    margin: auto;
+    padding: 25px 0 0;
+}
+
+.nav__menu {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.menu__item {
+    margin: 0 30px 0 0;
+    width: 42px;
+    height: 40px;
+}
+
+.menu__item img {
+    object-fit: contain;
+    width: 100%;
+    height: 100%;
+}
+.menu__link-user img {
+    border-radius: 50%;
+}
+
+.menu {
+    position: relative;
+}
+
+.menu__input {
+    position: absolute;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+    opacity: 0;
+}
+
+.menu__label {
+    position: relative;
+    cursor: pointer;
+}
+
+.mainpopup {
+    margin: 25px 100px 0 -200px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 345px;
+    height: 460px;
+    overflow-y: scroll;
+    flex-wrap: wrap;
+    background: #fbfbfb;
+    border: 5px solid black;
+    border-radius: 4px;
+}
+.mainpopup::before {
+    content: "";
+    position: absolute;
+    top: 40px;
+    left: 50%;
+    transform: translate(-45%, 0);
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 0 15px 20px 15px;
+    border-color: transparent transparent #000000 transparent;
+    display: none;
+}
+
+.mainpopup::-webkit-scrollbar {
+    width: 5px;
+}
+
+.mainpopup::-webkit-scrollbar-track {
+    background: #787a7ad8;
+}
+
+.mainpopup::-webkit-scrollbar-thumb {
+    background: #ec8179;
+}
+
+.mainpopup::-webkit-scrollbar-thumb:hover {
+    background: black;
+}
+
+.mainpopup__link {
+    width: 50px;
+    margin: 8px;
+    padding: 15px;
+    height: 50px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    box-sizing: content-box;
+    border-radius: 4px;
+}
+
+.mainpopup__link:hover {
+    border: 2px solid black;
```

Comment:
Set borders for the links above, and on hover just show them. Because now icons are displaced on hover due to the appearance of borders

Reply:
fixed it, thanks: 
.main-popup__link:hover {
    outline: 2px solid black;
    /* padding: 10px; */
}